src/test/kotlin/com/zepben/evolve/services/network/NetworkServiceComparatorTest.kt

```diff
@@ -1273,6 +1319,48 @@ internal class NetworkServiceComparatorTest : BaseServiceComparatorTest() {
         comparatorValidator.validateCompare(closedSwitch, openSwitch, expectModification = difference)
     }
 
+    @Test
+    internal fun compareSynchronousMachine() {
```

Comment:
todo: Not an error with this spot, but I can't add it where it belongs due to githubness.

You have missed the check of `UsagePoint.phaseCode`

Reply:
Added a check for it